modules/cudn-status-conditions.adoc

```diff
@@ -80,3 +80,123 @@ h|Message
+[cols="2a,2a,3a,3a",options="header"]
+|===
+
+|Condition type
+3+|Reason, Message, Resolution
+
+.6+|`invalid mtu`
+3+|One of the following messages is returned when the `mtu` is set incorrect:
+h|Reason
+h|Message
+h|Resolution
+
+|The `mtu` is set higher than `65536`.
+|`spec.network.localnet.mtu` in body should be less than or equal to `65536`.
+|You must the `mtu` lower than `65536`.
+
+|The `mtu` is set lower than `576`.
+|`spec.network.localnet.mtu` in body should be greater than or equal to `576`.
+|You must set the `mtu` higher than `576`
+
+|The `mtu` must be at least `1280` when using the IPv6 subnet.
+|`MTU should be greater than or equal to 1280 when an IPv6 subnet is used`
+|You must set the `mtu` higher than or equal to `1280` when you have an IPv6 subnet defined on your user-defined network configuration.
+|===
+
+.Invalid `PhysicalNetworkName` scenarios types for the `ClusterUserDefinedNetwork` CR
+[cols="2a,2a,3a,3a",options="header"]
+|===
+
+|Condition type
+3+|Reason, Message, Resolution
+
+.6+|`invalid PhysicalNetworkName`
+3+|One of the following messages is returned when the `PhysicalNetworkName` is set incorrect:
+h|Reason
+h|Message
+h|Resolution
+
+|The name of the physical network is not set.
+|`spec.network.localnet.physicalNetworkName: Required value`
+|You must set the `physicalNetworkName` field.
+
+|The name of the physical network does not meet minimum length requirements.
+|`spec.network.localnet.physicalNetworkName in body should be at least 1 chars long`
+|You must set physical network name to be at least one character in length.
+
+|The name of the physical network exceeds the maximum character limit of 253.
+|`spec.network.localnet.physicalNetworkName: Too long: may not be more than 253 bytes`
+|You must set physical network name to not exceed the 253 character in length.
+
+|The name of the physical network must not contain `,` or `:`.
+|`physicalNetworkName cannot contain "," or ":" characters`.
+|You must remove the `,` or `:` from the physical network name.
+|===
+
+.Invalid `role` scenarios types for the `ClusterUserDefinedNetwork` CR
+[cols="2a,2a,3a,3a",options="header"]
+|===
+
+|Condition type
+3+|Reason, Message, Resolution
+
+.6+|`role unset` or `role is primary`
+3+|One of the following messages is returned when the `spec.network.localnet.role` is set incorrect:
+h|Reason
+h|Message
+h|Resolution
+
+|The `role` field must be set for your localnet topology.
+|`spec.network.localnet.role: Required value`
+|You must set the `role` field.
+
+|`Primary` is not a supported value for the Localnet topology.
+|`spec.network.localnet.role: Unsupported value: "Primary": supported values: "Secondary"`
+|You must set the `role` for your Localnet topology to `Secondary`-the accepted value.
+|===
+
+.Invalid `subnets` and `ipam` scenarios types for the `ClusterUserDefinedNetwork` CR
+[cols="2a,2a,3a,3a,",options="header"]
+|===
+
+|Condition type
+3+|Reason, Message, Resolution
+
+.6+|`LocalnetInvalidSubnets`
```

Comment:
Something is wrong with this table definition the columns doesnt seem aliened, please have a look at table 6 "Message, Resolution.." headers line: 
https://github.com/openshift/openshift-docs/blob/4172305504eec2bd1142d427e3e0919bbe7cba9a/modules/cudn-status-conditions.adoc

Reply:
Yeah, I'm still trying to figure out the asciidoc markup on this one. Give me a little. This won't get past the docs team's review. Thank you

Reply:
fixed